frontend/src/app/shared/components/searchable-project-list/searchable-project-list.service.ts

```diff
@@ -97,6 +97,23 @@ export class SearchableProjectListService {
     }
   }
 
+  private handleKeyEnter(event:KeyboardEvent):void {
+    const focused = document.activeElement as HTMLElement|undefined;
+
+    // Do nothing if the current focus is within a list action
+    if (focused?.closest(projectListActionSelector)) {
+      return;
+    }
+
+    const first = document.querySelector<HTMLElement>(`${projectListActionSelector}:not(${projectListItemDisabled})`) as HTMLAnchorElement|undefined;
+
+    if (first) {
+      event.preventDefault();
+      first.focus();
+      window.location.href = first.href;
```

Comment:
This service is also used for `project-include`, where this fails because the elements are `label > checkbox` instead of an `a[href]`. It would probably be better to handle this case back in the component.

Reply:
This could also be fixed by calling `click()` on the element instead of setting the location, that way it just toggles the checkbox in project include.